### DISCUSSION TITLE
Help!!!!!!

### DISCUSSION THREAD

**Discussion by CarlosABH** [reflex-dev/reflex#2394] [General]
what's happening whith this or what am i doing wrong?
![Captura de pantalla 2023-12-24 202531](https://github.com/reflex-dev/reflex/assets/106352519/d1806aaa-c6f2-451c-b47c-f00bc0f15409)

**Comment by hasansezertasan** [comment]
What are you even trying to do? Can you help us understand the situation with a MRE and a more detailed explanation?

**Reply by CarlosABH** [reply]
this is when i started downloading the pip of reflex in my terminal
